src/manager.js

```diff
@@ -203,10 +203,12 @@ Manager.prototype = {
     off: function(events, handler) {
         var handlers = this.handlers;
         each(splitStr(events), function(event) {
-            if (!handler) {
-                delete handlers[event];
-            } else {
-                handlers[event].splice(inArray(handlers[event], handler), 1);
+            if (typeof handlers[event] !== 'undefined') {
```

Comment:
do we really need to check `typeof handlers[event] !== 'undefined'` here isn't it enough to just check `handlers[event]` ?